Header header
Array[] data
float32 resolution
float32 bbminx
float32 bbminy
float32 bbmaxx
float32 bbmaxy
float32 origo_index_i
float32 origo_index_j